spec/system/placements/schools/partner_providers/remove_a_provider_spec.rb

```diff
@@ -0,0 +1,208 @@
+    and_i_click_on_springfield_university_in_the_providers_list
+    then_i_see_the_show_page_for_springfield_university
+
+    # User clicks cancel link
+    when_i_click_on_delete_provider
+    and_i_am_asked_to_confirm_delete_springfield_university
+    and_i_click_on_cancel
+    then_i_see_the_provider_index_page
+
+    when_i_click_on_springfield_university_in_the_providers_list
+    then_i_see_the_show_page_for_springfield_university
+
+    when_i_click_on_delete_provider
+    and_i_confirm_by_clicking_the_delete_provider_button
+    then_i_see_success_message_for_removing_springfield_university
+    and_i_see_that_springfield_is_no_longer_listed_on_the_provider_index_page
+    and_a_notification_email_is_sent_to_springfield_university
+
+    given_ogdenville_university_is_not_onboarded_on_to_the_placements_service
+    when_i_click_on_ogdenville_university_in_the_providers_list
+    then_i_see_the_show_page_for_ogdenville_university
+
+    when_i_click_on_delete_provider
+    and_i_confirm_by_clicking_the_delete_provider_button
+    then_i_see_success_message_for_removing_ogdenville_university
+    and_i_see_that_ogdenville_is_no_longer_listed_on_the_provider_index_page
+    and_a_notification_email_is_not_sent_to_ogdenville_university
+  end
+
+  private
+
+  def given_providers_exist
+    @school = create(:placements_school)
+    @springfield_university = create(:placements_provider,
+                                     name: "Springfield University",
+                                     ukprn: "10101010",
+                                     urn: "101010",
+                                     email_address: "reception@springfield.ac.uk",
+                                     telephone: "0101 010 0101",
+                                     website: "http://www.springfield.ac.uk",
+                                     address1: "Undisclosed")
+    @springfield_university_user = create(:placements_user, providers: [@springfield_university])
+
+    @ogdenville_university = create(:placements_provider, name: "Ogdenville University", ukprn: "11111")
+    @ogdenville_university_user = create(:placements_user, providers: [@ogdenville_university])
+
+    @shelbyville_university = create(:placements_provider, name: "Shelbyville University", ukprn: "22222")
+
+    @springfield_partnership = create(:placements_partnership, school: @school, provider: @springfield_university)
+    @ogdenville_partnership = create(:placements_partnership, school: @school, provider: @ogdenville_university)
+    @shelbyville_partnership = create(:placements_partnership, school: @school, provider: @shelbyville_university)
+  end
+
+  def and_i_am_signed_in
+    given_i_am_signed_in_as_a_placements_user(organisations: [@school])
+  end
+
+  def when_i_am_on_the_providers_index_page
+    page.find(".app-primary-navigation__nav").click_on("Providers")
+    expect(page).to have_current_path(placements_school_partner_providers_path(@school))
```

Comment:
I had a conversation with @JamieCleare2525 about the use of `have_current_path` in system specs and I do not think that it belongs here! The other checks we do on the page are more than sufficient to prove we're in the expected place and the user does not specifically care about the URL.

Please can we remove all `have_current_path` matchers?